src/Helper/ConfigurationHelper.php

```diff
@@ -195,13 +196,17 @@ protected function loadFromDetectedFile()
             return $this->getDefault();
         }
         $this->detectedPath = $filePath;
-        return include $filePath;
+
```

Comment:
I don't see the need to store the result to a variable immediately before returning the value.

However, more importantly is that this won't handle a Yaml file (set via `$options['filename']`). I think this should be something like `return $this->getConfigurationArray($filePath);`